lms/djangoapps/teams/waffle.py

```diff
@@ -0,0 +1,26 @@
+"""
+Togglable settings for Teams behavior
+"""
+from django.conf import settings
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+# Course Waffle inherited from edx/edx-ora2
+WAFFLE_NAMESPACE = 'openresponseassessment'
```

Comment:
These are already constants somewhere in ORA, right? could we not just import them for safety's sake

Reply:
looks like in `lms/djangoapps/instructor_tasks/tasks_helper/misc.py` we import something directly from ORA so you probably can do it .... but you have a point, we probably want to avoid that